guides/Overview.md

```diff
@@ -1,9 +1,19 @@
 # Introduction
 
-Velo's APIs (previously Corvid) empower you to take full control of your site’s functionality. Use the APIs to interact with site elements, your site’s database content, Wix Apps, and external services. The APIs also give you access to information about your site, its users, and more.
+Welcome to Wix's Velo API Reference. This reference is your comprehensive guide to using Velo, Wix's powerful JavaScript-based development platform, to create dynamic and interactive web experiences. Velo empowers you to take full control of your site’s functionality. Use Velo APIs to fully customize your site, from frontend design to backend functionality.
 
-To use the APIs, you’ll need a working knowledge of JavaScript, including ES2020 features. The APIs include lots of code examples to help you [get started](tutorials/getting-started).
+The API reference provides detailed documentation and code examples for all APIs, functions and event handlers, and properties that you can use to interact with various aspects of your site. To use the APIs, you’ll need a working knowledge of JavaScript, including ES2020 features. 
```

Comment:
```suggestion
The API reference provides detailed documentation and code examples for all functions, event handlers, and properties that you can use to interact with various aspects of your site. To use the APIs, you’ll need a working knowledge of JavaScript, including ES2020 features. 
```
The commas and "and"s in the first sentence were a little confusing. Maybe this edit resolves the issue. (In any event I don't think APIs are something distinct from functions and event handlers here).